src/main/java/ether/JSON.java

```diff
@@ -0,0 +1,147 @@
+package ether;
+
+import io.left.rightmesh.id.MeshID;
+import io.left.rightmesh.mesh.MeshManager;
+import io.left.rightmesh.util.EtherUtility;
+import io.left.rightmesh.util.MeshUtility;
+import io.left.rightmesh.util.RightMeshException;
+import org.json.simple.JSONObject;
+
+import java.math.BigInteger;
+
+
+/**
+ * The JSON wrapper
+ */
+public final class JSON {
```

Comment:
One other thing - we've been using Protobuf for the data payloads we send around RightMesh as they're small and nice to work with. It would be worth considering replacing JSON with Protobuf in the future.

Reply:
This is the plan))

Reply:
Yes - we discussed this last week - since the microraiden-java project used json, it is easier if we continue to use it....like igor mentioned later on we will switch it to protobuf.